packages/vite/src/node/ssr/ssrModuleLoader.ts

```diff
@@ -28,7 +28,10 @@ type SSRModule = Record<string, any>
 
 const pendingModules = new Map<string, Promise<SSRModule>>()
 const pendingImports = new Map<string, string[]>()
-const importErrors = new WeakMap<Error, { importee: string }>()
+const errorStacks = new WeakMap<
+  Error,
+  WeakMap<string[], { importee: string }>
+>()
```

Comment:
It would be nice to avoid this nested weakmap. Is it possible to make this `WeakMap<string[], { importee: string }>` instead? Then we match by the `urlStack` instead of the `error`

Reply:
Ah that would be nice. Done. 👍